inc/savedsearch.class.php

```diff
@@ -334,8 +332,8 @@ function cleanDBonPurge() {
    /**
     * Print the saved search form
     *
-    * @param integer $ID      ID of the item
-    * @param array   $options Options:
+    * @param $ID         integer    ID of the item
```

Comment:
Yes, I've begin to change all comments already. The link I gave you are the GLPI CS rules.

Old rules were wrong, the API output was missing type, and this does not respect the PHPDOC guidelines.